packages/app/src/SVGIcons.tsx

```diff
@@ -248,3 +248,36 @@ export function S3Icon({ style, width }: IconProps) {
     </svg>
   );
 }
+
+export function Eye({ style, width }: IconProps) {
```

Comment:
I think we can use bootstrap icons instead https://icons.getbootstrap.com/?q=eye